bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+end
+```
+
+## Technical Information
+
+### Implemented Messages
+The table below shows what messages are implemented and to what extent. Transmit means we can build and transmit a packet of that type with relevant data. Decode means we can extract data into some meaningful form. All message types can be received, identified and the raw payloads displayed. Messages not identified in this table cannot be transmitted by the binding and can only be decoded as a raw payload.
+
+| Message               | Transmit | Decode           |Comments                                    |
+|-----------------------|:--------:|:----------------:|--------------------------------------------|
+|ACK                      | Y        | Y                |                                            |
+|PAIR PING                | N        | Y                |                                            |
+|PAIR PONG                | Y        | Y                |                                            |
+|SET GROUP ID             | Y        | Y                |                                            |
+|SET TEMPERATURE          | Y        | Y                | Allows setting of temperature of (wall)therm |
+|TIME INFO                | Y        | Y                |                                            |
+|WAKEUP                   | Y        | N                |                                            |
+|WALL THERMOSTAT CONTROL  | N        | Y                | Provides measured temp and set point       |
+|THERMOSTAT STATE         | N        | Y                | Provides battery/valvepos/temperature/thermostat set point |
+| WALL THERMOSTAT STATE   | N        | Y                | Provides battery/valvepos/temperature/thermostat set point |
+| PUSH BUTTON STATE       | N        | Y                | Auto maps to ON, Eco maps to OFF           |
+| ADD LINK PARTNER        | Y        | N                | Links a device with another                |
+| SET DISPLAY ACTUAL TEMP | Y        | Y                | Set a wall thermostat to show current measured or current setpoint temperature |
+
+## Message Sequences
+
+For situations such as the pairing where a whole sequences of messages is required the binding has implemented a message sequencing system. This allows the implementation of a state machine for the system to pass through as messages are passed back and forth.
+
+This will be documented in more detail in due course.
+
+## Planned Future Features
+
+These are in no particular priority and are simply ideas. They may not get implemented at all.
+
+1. If there is a pending SET_TEMPERATURE message in the queue and we receive a SET_TEMPERATURE from the thermostat we are waiting to send to then we should clear the message from the queue as it will be outdated.
+1. Explore how to avoid the queue getting too long due to lack of credits with many devices.
+1. Add ability to setup device groups which should help reduce lack of credit issue

```

Comment:
You could create GitHub issues to track these. Roadmaps shouldn't be part of the readme.

Reply:
Removed. It was copied from OH1.